packages/main/src/themes/sap_horizon_dark/DayPicker-parameters.css

```diff
@@ -34,5 +34,13 @@
 	--_ui5_daypicker_item_now_selected_between_border_radius: 0.1875rem;
 	--_ui5_daypicker_item_select_between_border: 1px solid var(--sapContent_Selected_ForegroundColor);
 	--_ui5_daypicker_item_now_inset: 0.375rem;
-	--_ui5-dp-item_withsecondtype_border: 0.1875rem;
+	--_ui5-dp-item_withsecondtype_border: 0.25rem;
```

Comment:
In some cases "dp" is used, in other "daypicker", I suggest use the shorter everywhere

Reply:
I agree with you. I propose that this happen in a separate change that I am preparing and I will give a reference to it here